src/applications/gi/components/search/InstitutionFilterForm.jsx

```diff
@@ -67,6 +78,25 @@ class InstitutionFilterForm extends React.Component {
     );
   };
 
+  renderCautionaryWarningsFilter = () => {
```

Comment:
can you break this out into its own component like how https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi/components/search/OnlineClassesFilter.jsx is ?